xep-0060.xml

```diff
@@ -49,6 +49,18 @@
   &pgmillard;
   &stpeter;
   &ralphm;
+  <revision>
+    <version>1.19.0</version>
+    <date>2020-08-16</date>
+    <initials>ps</initials>
+    <remark><p>Add missing 'item' key and 'retrieve' action to query type registry.</p></remark>
+  </revision>
+  <revision>
+    <version>1.18.1</version>
+    <date>2020-08-25</date>
+    <initials>lnjx</initials>
```

Comment:
@horazont Everywhere else in the xeps repo my initials have been 'lnj' (Linus N. Jahn)